### PR TITLE
Updates to aws-pgbench-postgresql

### PR DESCRIPTION
**Pull request by mw2q**
* updates related to previous terraform v1.5.x updates
* increased verbosity
* separating post-processing from execution tasks so failed post-processing can be done ad-hoc as long as the execution is successful

### CODE REVIEW

**Review by dougortiz — COMMENTED** [review]
Do we want to keep the verbosity at the 2nd level for the: 'aws-pgbench-postgresql/02_deploy/run.sh'?

**Comment by bryan-bar** [comment]
> Do we want to keep the verbosity at the 2nd level for the: 'aws-pgbench-postgresql/02_deploy/run.sh'?

There is an environment variable available we could use instead:
[ANSIBLE_VERBOSITY](https://docs.ansible.com/ansible/latest/reference_appendices/config.html#envvar-ANSIBLE_VERBOSITY)

**Comment by mw2q** [comment]
> > Do we want to keep the verbosity at the 2nd level for the: 'aws-pgbench-postgresql/02_deploy/run.sh'?
> 
> There is an environment variable available we could use instead: [ANSIBLE_VERBOSITY](https://docs.ansible.com/ansible/latest/reference_appendices/config.html#envvar-ANSIBLE_VERBOSITY)

Oh, TIL.  I'll remove it then.

**Comment by mw2q** [comment]
> > > Do we want to keep the verbosity at the 2nd level for the: 'aws-pgbench-postgresql/02_deploy/run.sh'?
> > 
> > 
> > There is an environment variable available we could use instead: [ANSIBLE_VERBOSITY](https://docs.ansible.com/ansible/latest/reference_appendices/config.html#envvar-ANSIBLE_VERBOSITY)
> 
> Oh, TIL. I'll remove it then.

rebased and forced push

**Review by dougortiz — APPROVED** [review]
LGTM

File: aws-pgbench-postgresql/01_provision/templates/inventory.yml.tftpl
Comment (bryan-bar) [inline aws-pgbench-postgresql/01_provision/templates/inventory.yml.tftpl:1]: This file can be removed as it seems we are using the default template from edb-terraform.

Comment (mw2q) [reply aws-pgbench-postgresql/01_provision/templates/inventory.yml.tftpl:1]: oops, will remove

File: aws-pgbench-postgresql/04_execute/run.sh [hunk 1]
```diff
@@ -26,6 +26,15 @@ do
 		-e "benchmark_duration=${BENCHMARK_DURATION}" \
 		-e "results_directory=${RESULTS_DIRECTORY}/report-data" \
 		${SCRIPT_DIR}/playbook-pgbench-run.yml
+
+	# Process results
+	ansible-playbook \
+		-vv \
```
Comment (bryan-bar) [inline aws-pgbench-postgresql/04_execute/run.sh:32]: leftover verbosity

Comment (mw2q) [reply aws-pgbench-postgresql/04_execute/run.sh:32]: oops, will fixup

**Comment by mw2q** [comment]
rebased and force push.

**Pull request merged**